src/node_api.h

```diff
@@ -31,7 +31,7 @@ struct uv_loop_s;  // Forward declaration.
     #define NAPI_EXTERN __declspec(dllexport)
   #endif
 #else
-  #define NAPI_EXTERN /* nothing */
+  #define NAPI_EXTERN __attribute((visibility("default")))
```

Comment:
Not sure whether it matters to you, but the `node.h` patch uses `__attribute__`, as does most of Node.js by itself (e.g. this would probably requested when upstreaming the change.)

Reply:
d'oh, this was just a typo. fixed :) good catch!